packages/instrumenter/src/transformers/babel-transformer.ts

```diff
@@ -18,6 +18,22 @@ import { AstTransformer } from './index.js';
 
 const { traverse } = babel;
 
+/**
+ *
+ * @param obj Represents the tag --useMutatationLevels
+ * @param propertyName Indicates which property should be selected
+ * @returns Returns the property of the desired mutator
+ */
+export function getPropertyByName<T, K extends keyof T>(obj: T, propertyName: K | string | undefined): string[] | undefined {
+  if (typeof propertyName !== 'string')
+    if (propertyName !== undefined) {
+      return obj[propertyName] as string[];
+    } else {
+      return undefined;
+    }
+  return undefined;
```

Comment:
can be simplified by writing
`if (typeof propertyName !== 'string') && propertyName !== undefined) {
  return obj[propertyName] as string[];
}
return undefined;
`